src/main/java/uk/gov/pay/adminusers/model/CompleteInviteRequest.java

```diff
@@ -5,19 +5,21 @@
 import io.swagger.v3.oas.annotations.media.ArraySchema;
 import io.swagger.v3.oas.annotations.media.Schema;
 
-import java.util.ArrayList;
+import java.util.Collections;
 import java.util.List;
 
 @JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
-public class InviteCompleteRequest {
+public class CompleteInviteRequest {
 
-    public static final String FIELD_GATEWAY_ACCOUNT_IDS = "gateway_account_ids";
-
-    @ArraySchema(schema = @Schema(implementation = String.class, example = "1,2",
+    @ArraySchema(schema = @Schema(example = "1",
             description = "gateway_account_ids that needs to be associated for the new service. Only applicable for invite type `service`"))
-    private List<String> gatewayAccountIds = new ArrayList<>();
-
-    public void setGatewayAccountIds(List<String> gatewayAccountIds) {
+    private List<String> gatewayAccountIds = Collections.emptyList();
```

Comment:
We'll be looking into this a little more when we change the registration journey on the frontend, so when we do it should be more obvious whether this is used or not. I think your suspicions are correct, so hopefully this can be made a lot simpler.